core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.ts

```diff
@@ -36,7 +37,31 @@ export class VisualizationPanelContentComponent implements OnInit, AfterViewInit
   }
 
   ngAfterViewInit() {
-    this.onClickGenerateChart();
+    if (this.data.chartType !== 'word cloud') {
```

Comment:
We should use a `switch` statement and expliclty list all the supported chart types. As I said in a review above, we need to expliclty write documentation about:
1. list of chart types we support
2. for each chart type, what data format the chart expect.
This needs to be documented in both frontend and backend. And each chart type should point to each other. For example, in frontend, and under the case `'word cloud' `, we should add a comment saying that it corresponds to  `WordCloudSink.java`

Reply:
I will add comments in both frontend and backend for visualization operators.